package/MDAnalysis/core/universe.py

```diff
@@ -992,6 +993,194 @@ def add_Segment(self, **attrs):
         self.segments = SegmentGroup(np.arange(self._topology.n_segments), self)
         # return the new segment
         return self.segments[segidx]
+    
+    def _add_TopologyObjects(self, object_type, values, types=None, guessed=False,
+                           order=None):
+        """Add new TopologyObjects to this Universe
+
+        Parameters
```

Comment:
Some parameters need more explanations:
What are valid values for `types` and `order`? Any restrictions?
What do they mean / what's their purpose?

What are the requirements for the different kinds of supported `values`?

Reply:
@lilyminium I think order only makes sense for bonds (ie single/double), so keep it in this "private" method and only expose it in the `add_Bonds`